cli/lockfile.rs

```diff
@@ -101,16 +102,36 @@ impl Lockfile {
   ) -> Result<Option<Lockfile>, AnyError> {
     if flags.no_lock {
       return Ok(None);
+    } else if let Some(config_file) = maybe_config_file {
+      if let Some(LockConfig::Bool(lock)) =
+        config_file.clone().to_lock_config()?
+      {
+        if !lock {
+          return Ok(None);
+        }
+      }
     }
```

Comment:
If a user specifies `flags.lock` then that should override what's in the configuration file.

So, this should probably be moved to a match on `match config_file.clone().to_lock_config()?` on line 120.

Sorry, didn't notice this until now.

Reply:
Done. Couldn't do it with a match because of the `return Ok(None)`.